ajax/libs/mui/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "mui",
+  "filename": "js/mui.min.js",
+  "description": "A effective front-end framework closest to the native app",
```

Comment:
@PeterDaveHello Then `A effective front-end framework allowing users to design components, like they are on the mobile application, running on browser`?